iot-hub/Samples/service/AzureSasCredentialAuthenticationSample/Parameters.cs

```diff
@@ -49,7 +49,14 @@ internal class Parameters
             's',
             "SharedAccessKey",
             Required = true,
-            HelpText = "The shared access key for IoT hub.")]
-        public string SharedAccessKey { get; set; }       
+            HelpText = "The shared access key for connecting to IoT hub.")]
+        public string SharedAccessKey { get; set; }
+
+        [Option(
+            'n',
+            "SharedAccessKeyName",
+            Required = true,
+            HelpText = "The shared access key name for the access key supplied.")]
```

Comment:
HelpText = "The shared access key name for the access key supplied. Eg. iothubowner, registryread etc..."